### DISCUSSION TITLE
Why is there no automatization for same line so I need to press enter 100 times?

### DISCUSSION THREAD

**Discussion by crododo** [community/community#90922] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Visual Studio

For example, I have 10 textboxes, let's call them txtBox1, txtBox2.... txtBox10
now if I wish to emty first box code would be:
txtBox1.text = ""
and then 
txtBox2.text = ""
and after I press enter it automaticly suggest txtBox3.text = ""

But sometimes I have examples like this with 100+ lines of almost identical code and I need to press 100 time enter + arrow (since TAB is not working with copilot) insted of just saying something like ' complete the same for the rest of textboxes

Is there any way around this?

**Comment by bonizario** [comment, marked as answer]
Whenever my TAB stops working with Copilot (I work in a significantly large codebase), I refresh VS Code and wait for the typescript stuff to load in.

You can also check the keybindings for your editor. In VS Code you can do it like this:

1. Press `CTRL+SHIFT+P` to open a VS Code panel that runs commands
2. Type: `Preferences: Open Keyboard Shortcuts`
3. Search for: `editor.action.inlineSuggest.commit`

If this helped you, consider marking my answer as the "Accepted answer" 😄
